### DISCUSSION TITLE
nsc operator sharing/config

### DISCUSSION THREAD

**Discussion by ccapuano-ssc** [nats-io/nats-server#3145] [Q&A]
Hello all,
We have successfully brought up a decentralized auth cluster of nats nodes using the "FULL" resolver, and everything is looking/working as we expect, however I have some questions about best practices around maintaining the local state directories of nsc and operations groups that would ultimately maintain them. I have tried setting up another terminal and "nsc pull"ing the config into a new environment to see if I could manage from there, but have not been successful. I am sure I can source control the "root" config inclusive of the operator, sys account and  keys, which I think I can then "nsc pull" to get the most up to date config, but that feels dirty. Is there a recommended approach to maintaining the state files in respect with the nats nodes so that multiple users of nsc can manage the accounts? I wasn't able to find any documentation pointing to best practices in this area.

Thanks in advance,
Chris

**Comment by acormier-maia** [comment]
@ccapuano-ssc what ended up working for you here?
